src/components/admin/personen/PersonLock.vue

```diff
@@ -17,13 +30,43 @@
     adminId: string;
     disabled: boolean;
   };
+  type PersonLockForm = {
+    selectedBefristung: string;
+  };
+  const vuetifyConfig = (state: {
+    errors: Array<string>;
+  }): { props: { error: boolean; 'error-messages': Array<string> } } => ({
+    props: {
+      error: !!state.errors.length,
+      'error-messages': state.errors,
+    },
+  });
+
+  const validationSchema: TypedSchema = toTypedSchema(
+    object({
+      selectedBefristung: string()
+        .required(t('admin.befristung.rules.required'))
+        .matches(DDMMYYYY, t('admin.befristung.rules.format'))
+        .test('notInPast', t('admin.befristung.rules.pastDateNotAllowed'), notInPast),
+    }),
+  );
+
+  const formContext: FormContext<PersonLockForm, PersonLockForm> = useForm<PersonLockForm>({
+    validationSchema,
+  });
+
+  const [selectedBefristung, selectedBefristungProps]: [
+    Ref<string | undefined>,
+    Ref<BaseFieldProps & { error: boolean; 'error-messages': Array<string> }>,
+  ] = formContext.defineField('selectedBefristung', vuetifyConfig);
```

Comment:
Das gehört meiner Sicht in die Parent und nicht im Child. Der Child sollte einfach die Values emitten, sowie die sind und der Parent wird sich um die Validierung kümmern. Das kannst du also in PersonDetails packen mit den anderen Schemas. Vermutlich müsstest du auch `selectedBefristung` renamen, weil wir das schon da haben. Vielleicht `selectedBefristungLock `oder so.

Reply:
Dann soll "isUnbefristet" auch zum Parent. Alle Validationschemas befinden sich zur Zeit im Parent. Die Child Komponenten waren bisher alle "dumb" und haben nur Events emitted. Ich würde ungern jetzt plötzlich, das so machen, dass ein Child Komponente Validierung macht nur für diesen Use case.